### PR TITLE
add github cli

### PR DESCRIPTION
**Pull request by danniel-lara**
Adicionei essa nova ferramenta cli do github

### CODE REVIEW

File: plugins/githubcli.plugin/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"icon": "githubcli",
+	"label": "GitHub Cli",
+	"description": "GitHub on the command lin",
```
Comment (MalteKiefer) [inline plugins/githubcli.plugin/metadata.json:4]: Please fix the typo

**Review by MalteKiefer — CHANGES_REQUESTED** [review]
Please fix the typo

File: plugins/githubcli.plugin/uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ -f "/etc/yum.repos.d/gh-cli.repo" ]; then
+  dnf -y remove gh
```
Comment (kwizart) [inline plugins/githubcli.plugin/uninstall.sh:4]: It's possible that gh-cli.repo to be removed by hand. So it's unrelated to the availability of the gh package.
Please move the dnf remove out of the gh-cli.repo test condition.

**Comment by kwizart** [comment]
@danniel-lara Can you fix the two points ?

**Pull request merged**